decidim-admin/app/controllers/decidim/admin/scopes_controller.rb

```diff
@@ -79,6 +75,10 @@ def parent_scope
         @parent_scope ||= @scope ? @scope.parent : collection.find_by_id(params[:scope_id])
       end
 
+      def parent_scope_children
+        @parent_scope_children ||= parent_scope ? parent_scope.children : collection.top_level
+      end
+
```

Comment:
I agree with the name change for current `collection` method. 
I'm thinking to rename `parent_scope_children` as `children_scopes`. So, we'd have `parent_scope`, `scope` and `children_scopes`:
* `scope` is `nil` on root view and current scope in others
* `parent_scope` is `nil` on root and top level scopes views
* `children_scopes` has top levels scopes on root view and `scope` children in others

What do you think?